### ISSUE TITLE
Should Relighting gt Loss be added in to the loss function？

### ISSUE THREAD

**Issue by YaelCassini** [LansburyCH/relightable-nr#9]
Sorry to bother you, I changed my own data for the test of the relightable-nr model. My data is face mesh, and I have replaced the tex  with the diffuse map of the face, but in the training results, the effect of relighting is not very good, the face looks like a metal-like material. After checking the source code, I found The loss of relighting results and relighting ground truth is not added to the loss function, so I wonder if this is the cause of this problem?

**Comment by LansburyCH** [comment]
Hi YaelCassini,

The original problem setting of our paper takes only images under a same lighting as input, so there is no relighting groundtruth. However, if there is relighting supervision, then you could consider adding relighting losses. For the metal-like appearance, I'm not very certain about this, but you can try increasing the number of sampled directions for the diffuse part and/or removing the specular part.

**Comment by YaelCassini** [comment]
Thank you very much, I'll try your suggestion then.
